### DISCUSSION TITLE
Module that automatically sell items based on quality

### DISCUSSION THREAD

**Discussion by silviu20092** [azerothcore/azerothcore-wotlk#18558] [Modules]
Hello, due to constant need of selling junk items to vendors, I have created a module that automates the selling of items based on their quality. The module exposes a single command **.sell #quality** (example .sell grey - will sell all grey (poor quality) items, .sell blue - will sell all rare quality items etc). The player will get the right amount of gold after using the command and the sold items will appear in the vendor's buyback list. It's exactly like selling the items to a vendor, except you don't need one!
This is the github link to my repo: https://github.com/silviu20092/mod-sell-items
I don't really know how to add it as an "official" mod yet.

**Comment by Scylune** [comment]
Hello, thanks !
I'll try it soon ^^

Edit : Tested with grey/green/blue/purple, works fine.
I'll keep it ! Thanks.

**Comment by sudlud** [comment]
there's also https://github.com/noisiver/mod-junk-to-gold available for grey items only
